modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/ManageTranslations.path

```diff
@@ -21,6 +21,11 @@
 	<td>//tr//td[contains(text(),'${key_language}')]/following-sibling::td//button[@*[contains(.,'Delete')]]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>LANGUAGE_STATUS</td>
+	<td>//tr//td[contains(text(),'${key_language}')]/following-sibling::td//span[text()='${key_text}']</td>
```

Comment:
I think it's better to use ${key_status} instead of ${key_text} as it's more clear for us to know what's the variable for.